index.html

```diff
@@ -0,0 +1,203 @@
+    </ul>
+    <div class="btn-div">
+      <button class="btn" type = "submit"> See Project</button>
+    </div>
+  </div>
+  <div class="second-sec-div">
+    <div>
+      <img class="proj-img" src="images/Snapshoot Portfolio (1).png" alt="picture" >
+    </div>
+    <div>
+      <h4 class="proj-title">Multi-Post <br>
+        Stories </h4>
+    </div>
+    <div class="proj-intro">
+      <span class="intro-1">CANOPY</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-2">Back end Dev</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-3">2015</span>
+    </div>
+    <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="proj-lan">
+      <li class="lan">HTML</li>
+      <li class="lan"> CSS</li>
+      <li class="lan">Javascript</li>
+    </ul>
+    <div class="btn-div">
+      <button class="btn" type = "submit"> See Project</button>
+    </div>
+  </div>
+  <div class="second-sec-div">
+    <div>
+      <img class="proj-img" src="images/Snapshoot Portfolio (2).png" alt="picture" >
+    </div>
+    <div>
+      <h5 class="proj-title">Tonic</h5>
+    </div>
+    <div class="proj-intro">
+      <span class="intro-1">CANOPY</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-2">Back end Dev</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-3">2015</span>
+    </div>
+    <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="proj-lan">
+      <li class="lan">HTML</li>
+      <li class="lan"> CSS</li>
+      <li class="lan">Javascript</li>
+    </ul>
+    <div class="btn-div">
+      <button class="btn" type = "submit"> See Project</button>
+    </div>
+  </div>
+  <div class="second-sec-div">
+    <div>
+      <img class="proj-img" src="images/Snapshoot Portfolio (3).png" alt="picture" >
+    </div>
+    <div>
+      <h6 class="proj-title">Multi-Post <br>
+        Stories</h6>
+    </div>
+    <div class="proj-intro">
+      <span class="intro-1">CANOPY</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-2">Back end Dev</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-3">2015</span>
+    </div>
+    <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="proj-lan">
+      <li class="lan">HTML</li>
+      <li class="lan"> CSS</li>
+      <li class="lan">Javascript</li>
+    </ul>
+    <div class="btn-div">
+      <button class="btn" type = "submit"> See Project</button>
+    </div>
+  </div>
+  </div>
+</section>
+<section class="sec-3">
+  <div class="section3-div">
+  <div class="third-section" >
+    <div>
+    <h1 class="title">
+      About Myself
+    </h1>
```

Comment:
- [ ] Please kindly ensure to use any of the `hx` tag as pointed out earlier, this will enhance your app SEO. 👍